h2/src/test/org/h2/test/TestAll.java

```diff
@@ -929,6 +931,7 @@ private void testAdditional() {
         addTest(new TestFileLockProcess());
         addTest(new TestDefrag());
         addTest(new TestTools());
+        addTest(new TestIgnoreCatalogs());
```

Comment:
It should be invoked from `test()` instead.

Reply:
done